### PR TITLE
Switch to Pipenv

### PR DESCRIPTION
**Pull request by ezwang**
- Also switch from `npm` to `yarn`.
- Also switch from TravisCI to CircleCI.
- Add script to parse fixture dumps and setup fixtures for development.
- Before deploying, ensure that production server is set up with Pipenv.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,8 @@ cache:
     directories:
         - "$HOME/.cache/pip"
 install:
-    pip install flake8 -r requirements.txt
+    - pip install pipenv
+    - pipenv install --dev
 script:
-    flake8 --max-line-length=250 --exclude=*/migrations/* api apps
```
Comment (sigmachirality) [inline .travis.yml]: are we paying more attention to line length now?

File: lib/api/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-from .api import *
+from .api import *  # noqa
```
Comment (sigmachirality) [inline lib/api/__init__.py]: what's this comment

File: settings.py [hunk 3]
```diff
@@ -156,7 +156,7 @@
 
 # Used for Django debug toolbar (or use debugsqlshell)
 try:
-    import debug_toolbar
+    import debug_toolbar  # noqa
```
Comment (sigmachirality) [inline settings.py]: is this a lint flag or smth

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,9 @@
+[flake8]
+max-line-length = 250
```
Comment (sigmachirality) [inline setup.cfg]: oh lol I see we moved the lines stuff here

Comment (ezwang) [reply setup.cfg]: Yup, this way it also runs with the right config when you do `flake8` without arguments.

**Pull request merged**